src/macros.rs

```diff
@@ -1,18 +1,15 @@
-
```

Comment:
If all these changes are the results of latest rustfmt, it would be useful to just run `cargo fmt` on current master branch and just submit that as its own PR.

Reply:
Yeah, that was unintentional, whenever I saved it would automatically reformat it, and I might have done a git add . I reverted it